docs/integrations/keras_integration.rst

```diff
@@ -0,0 +1,597 @@
+
+   data_ham.to_csv("data_ham.csv")
+   data_spam.to_csv("data_spam.csv")
+
+   #Log the generated artifact in Neptune:
+   neptune.log_artifact("data_ham.csv")
+   neptune.log_artifact("data_spam.csv")
+
+
+You can find the dataset in the artifact |page| on Neptune.
+Next, let's make some word clouds to better understand the dataset. Note that you can also log these charts on Neptune as well.
+
+.. |page| raw:: html
+
+    <a href="https://ui.neptune.ai/risingodegua/spam-classifier/e/SPAM-2/artifacts" target="_blank">page</a>
+
+
+.. code-block:: python3
+
+   def show_wordcloud(df, title):
+      text = ' '.join(df['text'].astype(str).tolist())
+      stopwords = set(wordcloud.STOPWORDS)
+      fig_wordcloud = wordcloud.WordCloud(stopwords=stopwords,background_color='lightgrey',
+                     colormap='viridis', width=800, height=600).generate(text)
+   
+      plt.figure(figsize=(10,7), frameon=True)
+      plt.imshow(fig_wordcloud) 
+      plt.axis('off')
+      plt.title(title, fontsize=20 )
+      plt.savefig(f"{title}.png")
+      plt.show()
+      neptune.log_image(title, f'{title}.png') # log the image
+
+
+The function above will use the wordcloud package to generate wordclouds of the different common words found in each category. 
+First, let’s make a wordcloud for Non-Spam (Ham) messages:
+
+.. code-block:: python3
+
+   show_wordcloud(data_ham, "Ham messages")
+
+.. image:: ../_static/images/integrations/keras-integration/kix.fu41vu4dfwtu.png
+
+
+
+Next, let’s plot one for Spam messages:
+
+
+.. code-block:: python3
+
+   show_wordcloud(data_spam, "Spam messages")
+
+.. image:: ../_static/images/integrations/keras-integration/kix.wx6tj72zb4n.png
+
+
+
+|Neptune link| to view the saved images.
+Next, you'll split the dataset into train and test set:
+
+.. |Neptune link| raw:: html
+
+    <a href="https://ui.neptune.ai/risingodegua/spam-classifier/e/SPAM-2/logs" target="_blank">Neptune link</a>
+
+
+.. code-block:: python3
+
+   from sklearn.model_selection import train_test_split
+
+.. code-block:: python3
+
+   X = data['text'].values
+   y = data['label'].values
+   X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.20, random_state=42)
+
+   ##Save in properties
+   neptune.set_property('test_split_percent', 0.20)
+   neptune.set_property('data_split_random_state', 42)
+
+
+Next, you’ll import keras from tensorflow and also some pre-processing functions:
+
+.. code-block:: python3
+
+   from tensorflow.keras.preprocessing.sequence import pad_sequences
+   from tensorflow.keras.preprocessing.text import Tokenizer
+   from tensorflow.keras.models import Sequential
+   from tensorflow.keras.layers import Dense
+   from tensorflow.keras.layers import Dropout
+   from tensorflow.keras.layers import Flatten
+   from tensorflow.keras.layers import Embedding
+   from tensorflow.keras.callbacks import EarlyStopping
+
+
+
+Next, let’s do some basic text pre-processing since the dataset is made up of SMS. 
```

Comment:
Again this makes it hard for the reader to focus on the integration. 

Remember this page should educate people on the neptune-keras integration. 
... but a big part of this tutorial is about other ML/DS tasks (important for sure but not for this page).